packages/website/pages/docs/usage/configuration.mdx

```diff
@@ -71,6 +71,21 @@ t('ordered', {orderDate: parseISO('2020-11-20T10:36:01.516Z')});
 t('latitude', {latitude: 47.414329182});
 ```
 
+## Global Rich Text Elements
+To achieve consistent usage and styling of rich text elements, you can define a set of global elements and pass them to the provider.
+If you define an element locally it will override the global default.
+
+```js
+<NextIntlProvider
+  ...
+  defaultRichTextElements={{
+    b: (value) => <strong>{value}</strong>
```

Comment:
I'm calling the argument "children" in [the rich text section](https://next-intl-git-fork-maxwoedl-main-amann.vercel.app/docs/usage/messages#rich-text). Can you adapt the wording here and possibly add brackets around the argument in the rich text section for consistency? Your version with brackets is better.

Can you also cross reference this section from the rich text docs?